src/main/resources/mybatis/mappers/LanguageMapper.xml

```diff
@@ -17,19 +17,28 @@
     </select>
 
     <insert id="insert" useGeneratedKeys="true" keyColumn="id" keyProperty="id">
-        INSERT INTO languages (name, has_localization)
-        VALUES (#{name}, #{hasLocalization})
+        INSERT INTO languages (name, has_localization, icon)
+        VALUES (#{name}, #{hasLocalization}, #{icon})
```

Comment:
Shouldn't it be here is_default as well? Anyone who can add a language or update it should be able to choose the default language. We need is_default to be updatable.